BenadrylAutomation/src/main/java/com/jnj/pageobjects/HomePage.java

```diff
@@ -0,0 +1,198 @@
+		Action.performActionwithExtentInfoLog(this.learnMoreBtn, "click",
+				"Clicking on : Learn More" );
+		Action.verifyPageUrl(expectedUrl);
+		extentInfoLog("Successfully verified URL!","");
+	}
+
+	/**
+	 * Verifies the Our Products section by checking the displayed section, heading,
+	 * clicking on a specific product, and verifying its URL.
+	 *
+	 * @param index The index of the Our Products section.
+	 * @param expectedHeading The expected heading of the Our Products section.
+	 * @param expectedUrl The expected URL after clicking on "All Products".
+	 * @param productName The name of the specific product to click on.
+	 * @param pdctURL The expected URL after clicking on the specific product.
+	 */
+	public void verifyOurProducts(String index, String expectedHeading, String expectedUrl, String productName, String pdctURL) {
+		extentInfoLog("Section is displayed : ",Action.isDisplayed(driver,this.getSection(index)));
+		String actualHeading = this.getSection(index).getText();
+		Assert.assertEquals(actualHeading, expectedHeading);
+		extentInfoLog("Heading is displayed : ",actualHeading);
+		Action.performActionwithExtentInfoLog(this.allPdctBtn, "click",
+				"Clicking on : " + allPdctBtn.getText());
+		Action.verifyPageUrl(expectedUrl);
+		extentInfoLog("URL Verified!","");
+		Action.navigateBack();	
+		Action.performActionwithExtentInfoLog(this.getPdct(productName), "click",
+				"Clicking on : " + getPdct(productName).getText());
+		Action.verifyPageUrl(pdctURL);
+		Action.navigateBack();
+	}
+
+	/**
+	 * Verifies the Best Seller section by checking the displayed section, heading,
+	 * and the presence of products within the Best Seller section.
+	 *
+	 * @param expectedHeading The expected heading of the Best Seller section.
+	 */
+	public void verifyBestSeller(String expectedHeading) {
+		extentInfoLog("Bestseller Section is displayed : ",Action.isDisplayed(driver,this.bestSellerHeading));
+		String actualHeading = this.bestSellerHeading.getText();
+		Assert.assertEquals(actualHeading, expectedHeading);
+		extentInfoLog("Heading is displayed : ",actualHeading);
+		extentInfoLog("Products are displayed : ",Action.isDisplayed(driver,this.bestSellerPdct));
+	}
+
+	/**
+	 * Verifies the Related Content section by checking the displayed section, heading,
+	 * clicking on a specific content card, and verifying the resulting page URL.
+	 *
+	 * @param expectedHeading The expected heading of the Related Content section.
+	 * @param index The index of the content card to click on.
+	 * @param expectedUrl The expected URL after clicking on the content card.
+	 */
+	public void verifyRelatedContent(String expectedHeading, String index, String expectedUrl) {
+		extentInfoLog("Related Content Section is displayed : ",Action.isDisplayed(driver,this.relatedContent));
+		String actualHeading = this.relatedContent.getText();
+		Assert.assertEquals(actualHeading, expectedHeading);
+		System.out.println(this.getRelatedContentCard(index));
```

Comment:
@RashiTiwari21 Please remove system log. If required use the extentInfo log instead

Reply:
fixed

Reply:
Verified @RashiTiwari21